tests/e2e/manifest/istio-gateway.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)